sites/partners/__tests__/testHelpers.ts

```diff
@@ -0,0 +1,314 @@
+import {
```

Comment:
Both of these items can be for another PR:

- It may make sense to move this to `shared-helpers`, since the public site unit tests will also benefit from everything that is in here.

- It may make it easier for us to maintain if we have structured test data, like `minimum listing`, `listing with units`, `listing with photo`, etc. And have all relationships split out like users and units, so we have one place to update if we change the models.

Reply:
Good call out. I'll create a story for that. The mocked object could be cleaned up in general